src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,44 +1,72 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import { getUser } from '../../api';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
+
+type Pros = {
+  selectedTodo: Todo,
+  selectMethod: (value: null) => void,
+};
+
+export const TodoModal: React.FC<Pros> = ({
+  selectedTodo,
+  selectMethod,
+}) => {
+  const [user, setUser] = useState<User | null>(null);
+
+  const getUserFromServer = async () => {
+    try {
+      setUser(await getUser(selectedTodo.userId));
+    } catch (error) {
+      throw new Error('Error user loading');
```

Comment:
In the catch block, you are throwing a new error but there is no error handling mechanism in the component. It would be better to have a state to hold the error and display it to the user.